src/main/java/TaskManager.java

```diff
@@ -2,18 +2,32 @@
 
 public class TaskManager {
     private final String BOT_NAME = "[iWish]: ";
+    public String[] taskList = new String[100];
+    public int trackList = 0;
 
-    public void echoInput() {
+    public void startUp() {
```

Comment:
The naming of this function could probably be improved since startUp is a bit confusing...

Reply:
Also, this part is a bit too messy. You could probably considering using switch case instead of if else, and wrap up printing stuff into other class